script/mbx

```diff
@@ -224,7 +234,10 @@ def latex_image_conversion(xml_source, xmlid_root, data_dir, dest_dir, outformat
             if (outformat == 'svg' or outformat == 'all'):
                 pdfsvg_executable = get_executable(config, 'pdfsvg')
                 _debug("pdfsvg executable: {}".format(pdfsvg_executable))
-                svg_cmd = [pdfsvg_executable, latex_image_pdf, latex_image_svg]
+                if plat == "Windows":
+                    svg_cmd = [pdfsvg_executable, "-density", "300", latex_image_pdf, latex_image_svg]
```

Comment:
The images were very blurry since the default density for ImageMagick's `convert` is something crazy low like 72dpi.